src/components/30-organisms/footer/child.scss

```diff
@@ -9,7 +8,12 @@ axa-footer {
     }
   }
 
-  h1, h2, h3, h4, h5, h6 {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
```

Comment:
This stuff is a bit of a joke, but we have little code like this and it is not too bad imho.

Reply:
no, re-format

Reply:
@markus-walther